File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1013_basico.js
import { question } from "readline-sync";

function main() {
    const valores = get_number();

    const Nvalores = valores.split(` `);
    let A = parseInt(Nvalores[0]);
    let B = parseInt(Nvalores[1]);
    let C = parseInt(Nvalores[2]);

    let maiorAB = Eh_Maior(A, B);
    let maiorTotal = Eh_Maior(maiorAB, C)

    print(`${maiorTotal} eh o maior`);
}

function get_number() {
    return (question(`Numero: `));
}

function print(texto) {
    return console.log(texto);
}

function Eh_Maior(A, B) {
    if (A > B) {
        return A
    }
    return B
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1013_basico.py
# valores = map(int, input().split())
# m = max(valores)
# print(f"{m} eh o maior")

def main():
    v1, v2, v3 = map(int, input().split())
    m = ehmaior(v1, v2)
    m = ehmaior(m, v3)
    print(f"{m} eh o maior")

def ehmaior(n1, n2):
    maior = int((n1 + n2 + abs(n1 - n2)) / 2)
    return maior

main()